contributors/devel/sig-node/e2e-node-tests.md

```diff
@@ -244,13 +257,15 @@ make test_e2e_node TEST_ARGS="--cgroups-per-qos=true"
 # Notes on tests run by the Kubernetes project during pre-, post- submit.
 
 The node e2e tests are run by the PR builder for each Pull Request and the results published at
-the bottom of the comments section.  To re-run just the node e2e tests from the PR builder add the comment
-`@k8s-bot node e2e test this issue: #<Flake-Issue-Number or IGNORE>` and **include a link to the test
+the bottom of the comments section. To re-run just the node e2e tests from the PR builder add the comment
+`/retest pull-kubernetes-node-e2e` and **include a link to the test
 failure logs if caused by a flake.**
 
-The PR builder runs tests against the images listed in [jenkins-pull.properties](https://git.k8s.io/kubernetes/test/e2e_node/jenkins/jenkins-pull.properties)
+The PR builder runs tests against the images listed in [https://github.com/kubernetes/test-infra/blob/master/jobs/e2e_node/image-config.yaml].
 
-The post submit tests run against the images listed in [jenkins-ci.properties](https://git.k8s.io/kubernetes/test/e2e_node/jenkins/jenkins-ci.properties)
+Other [prow jobs](https://github.com/kubernetes/test-infra/blob/master/prow/jobs.md)
```

Comment:
It may be worth linking where the node e2e jobs are defined, but I won't block if you decide to leave this as-is
```suggestion
Other [node e2e prow jobs](https://github.com/kubernetes/test-infra/tree/master/config/jobs/kubernetes/sig-node)
```

Reply:
AGHH, that's terrifying. At no point do we mention what code is being run. Not sure down here is the best place, but I might jam it in here because it is close in context.